model/job_reference.go

```diff
@@ -11,7 +11,8 @@ type JobReference struct {
 	Name                string                     `yaml:"name"`    // The name of the job
 	ReleaseName         string                     `yaml:"release"` // The release the job comes from
 	ExportedProviders   map[string]JobProvidesInfo `yaml:"provides"`
-	ResolvedConsumers   map[string]JobConsumesInfo `yaml:"consumes"`
+	ResolvedConsumers   map[string]JobConsumesInfo `yaml:"consumes"`    // Instance groups that this job links to & requires
```

Comment:
How about aligning the struct field label with the yaml field label? It is a bit confusing one being `...Consumers` and the other `consumes`.

Reply:
I feel like that should be done separately (this was just a comment change; refactoring would be a bigger change that should get its own PR).  I might still be convinced though!

Reply:
It is ok to do in a separate PR, so we don't mix things up! Could we do right after?

Reply:
Card created to tackle this.